crates/holochain_p2p/tests/tests/mod.rs

```diff
@@ -0,0 +1,4 @@
+mod local_agent;
+mod node_messaging;
+mod op_store;
+mod peer_meta_store;
```

Comment:
rust links a whole new test binary for every top-level integration test... and linking is very slow. I've moved all the integration tests into sub-modules per the pattern we've used in other crates to get that time back.

Reply:
Yep that's good. I prefer being able to filter these tests and only build/run what I'm working on but as soon as we actually depend on C code that ages to link I get the change :)